src/sql/workbench/contrib/query/browser/queryResultsView.ts

```diff
@@ -237,6 +237,16 @@ export class QueryResultsView extends Disposable {
 			if (runner.messages.some(v => v.isError)) {
 				this._panelView.showTab(this.messagesTab.identifier);
 			}
+			// Currently we only need to support visualization options for the first result set.
```

Comment:
do we really only need this for the first result set?  is that what KustoExplorer does?

Reply:
yes, I confirmed this with Rajesh, in a given query window, you can only run one query at a time.